### PR TITLE
docs: add faucet instructions

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Add instructions for using test network faucets.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct `docs:` prefix in the PR title
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the [documentation writing guidelines](https://github.com/cosmos/cosmos-sdk/blob/master/docs/DOC_WRITING_GUIDELINES.md)
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct `docs:` prefix in the PR title
- [ ] confirmed all author checklist items have been addressed 
- [ ] confirmed that this PR only changes documentation
- [ ] reviewed content for consistency
- [ ] reviewed content for thoroughness
- [ ] reviewed content for spelling and grammar
- [ ] tested instructions (if applicable)

### CODE REVIEW

File: docs/getting-started/live-networks.md [hunk 1]
```diff
@@ -86,4 +86,20 @@ You can check the configuration by running the following command:
...
+*For Redwood Testnet:*
+
+```bash
+curl http://209.182.218.23:8000/faucet/<account_address>
+```
+
+*For Hambach Testnet:*
+
+```bash
+curl http://147.182.238.116:8000/faucet/<account_address>
```
Comment (ryanchristo) [inline docs/getting-started/live-networks.md:104]: @kaustubhkapatral I realize this is not working at the moment but is the plan to reuse the same address once we get hambach back online?

Comment (kaustubhkapatral) [reply docs/getting-started/live-networks.md:104]: Faucet was moved from that to this server `http://185.181.11.150:8000/faucet`. I can reuse this server when we restart hambach.

Comment (ryanchristo) [reply docs/getting-started/live-networks.md:104]:
```suggestion
curl http://185.181.11.150:8000/faucet/<account_address>
```

Comment (kaustubhkapatral) [reply docs/getting-started/live-networks.md]: @ryanchristo I've assigned domains to sentries of hambach and redwood which also hosts the faucet.

http://redwood-sentry.vitwit.com:8000/faucet/<account_address>

http://hambach-sentry.vitwit.com:8000/faucet/<account_address>

**Review by anilCSE — APPROVED** [review]

**Review by kaustubhkapatral — APPROVED** [review]

**Pull request merged**